components/backdrop/backdrop.js

```diff
@@ -75,6 +75,12 @@ class Backdrop extends LitElement {
 		this._state = null;
 	}
 
+	disconnectedCallback() {
+		// allow body scrolling if backdrop is removed from the DOM
```

Comment:
Also adding a fix for body scrolling here to prevent any future issues where the backdrop is unexpectedly removed from the DOM and scrolling is blocked. This will only release the scroll if it had been blocked by this particular backdrop (ie. shouldn't affect any other backdrop instances on a page, it can only access its personal scroll key, if it exists)

Reply:
I would agree. If the backdrop gets disconnected we would want to reveal any elements hidden by it.